sirepo/package_data/static/js/srw.js

```diff
@@ -1950,6 +1950,14 @@ SIREPO.app.directive('simulationStatusPanel', function(appState, beamlineService
             };
 
             $scope.startSimulation = function() {
+                // The available jobRunModes can change. Default to parallel if
+                // the current jobRunMode doesn't exist
+                if (
+                    appState.models[$scope.simState.model].jobRunMode
+                        in authState.jobRunModeMap === false
+                ) {
+                    appState.models[$scope.simState.model].jobRunMode = 'parallel';
+                }
                 if ($scope.model == 'multiElectronAnimation') {
                     appState.saveChanges($scope.simState.model);
```

Comment:
I think there's a double save going on. May be able to delete this save.

Reply:
I'm not sure but I'm leaning towards the double save being necessary.`getReportTitle()` is using appState which possibly implies the appState needs to be updated with these values for it to work. Also, the call to `saveChanges` within `saveAndRunSimulation` uses `simulation` as the name whereas the call to `saveChanges` here uses `multiElectronAnimation` as the name. I'm not familiar enough with the code to have a positive answer. Should I do a dive and investigate?